fixtures/get-platform-proxy/tests/get-platform-proxy.env.test.ts

```diff
@@ -164,8 +193,6 @@ describe("getPlatformProxy - bindings", () => {
 		await dispose();
 	});
 
-	// Note: the following test is skipped due to flakiness caused by the local registry not working reliably
-	//       when we run all our fixtures together (possibly because of race condition issues)
```

Comment:
The test below needs to still be skipped at it doesn't seem to be passing 😢

It is also not passing on the latest stable wrangler release so it's likely an unrelated regression 😓

Reply:
This is being addressed in https://github.com/cloudflare/workers-sdk/pull/5669 🙂